tests/driver_dfplayer/Makefile

```diff
@@ -10,3 +10,8 @@ include $(RIOTBASE)/Makefile.include
 ifneq (,$(BUSY_PIN))
   CFLAGS += '-DDFPLAYER_PARAM_BUSY_PIN=$(BUSY_PIN)'
 endif
+
+ifneq (,$(filter arch_avr8,$(FEATURES_USED)))
+  # no strerror() in avrlibc; which makes a lot of sense on a Harvard machine
+  CFLAGS += -DDFPLAYER_NO_STRERROR
+endif
```

Comment:
That should probably go into `Makefile.dep` instead.

Reply:
I would prefer this in Makefile.include. Now that the order of inclusion is merged, let's try to do things right.

Reply:
Indeed. I first thought `strerror()` would be used in the `test`, not the shell command, at which point it made sense to put it only locally in the tests `Makefile`. I forgot to reconsider that decision

Reply:
> I would prefer this in Makefile.include. Now that the order of inclusion is merged, let's try to do things right.

:+1:. I will rebase (without squashing) to make local testing easier